Sources/Sentry/SentryBaggage.m

```diff
@@ -0,0 +1,59 @@
+#import "SentryBaggage.h"
+#import "SentryDsn.h"
+#import "SentryLog.h"
+#import "SentryOptions+Private.h"
+#import "SentryScope+Private.h"
+#import "SentrySerialization.h"
+#import "SentryTraceContext.h"
+#import "SentryTracer.h"
+#import "SentryUser.h"
+
+@implementation SentryBaggage
+
+- (instancetype)initWithTraceId:(SentryId *)traceId
+                      publicKey:(NSString *)publicKey
+                    releaseName:(nullable NSString *)releaseName
+                    environment:(nullable NSString *)environment
+                    transaction:(nullable NSString *)transaction
+                         userId:(nullable NSString *)userId
+                    userSegment:(nullable NSString *)userSegment
+{
+
+    if (self = [super init]) {
+        _traceId = traceId;
+        _publicKey = publicKey;
+        _releaseName = releaseName;
+        _environment = environment;
+        _transaction = transaction;
+        _userId = userId;
+        _userSegment = userSegment;
+    }
+
+    return self;
+}
+
+- (nullable NSString *)toHTTPHeader
```

Comment:
`m`: Shouldn't this be nonnull? You can only create a `SentryBaggage` with at least traceId and publicKey

Reply:
Yes. I don't think is possible to return null here.